app/frontend/src/app/layout.tsx

```diff
@@ -0,0 +1,67 @@
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import "./globals.css";
+import localFont from 'next/font/local'
+import Image from "next/image";
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar" /*Importinh avatar from shadcn */
+import Footerwave from "../../public/Footer_Wave.svg"; /*Imporrting the wave svg */
+
+const inter = Inter({ subsets: ["latin"] });
```

Comment:
Remove fonts that you are sure you will not be using, or move the to a separate file (Inter font)